docs/guides/mock-responses.md

```diff
@@ -153,3 +153,7 @@ When we call `curl http://127.0.0.1:4010/pets/123?__dynamic=true`, the operation
 The more descriptive your description is, the better job Prism can do at creating a mock response.
 
 _**Tip:** If your team needs help creating better quality API description documents, take a look at [Spectral](https://stoplight.io/spectral/). You could enforce the use of `example` properties, or similar._
+
+### Mocking OpenApi 3 callbacks
```

Comment:
```suggestion
### Mocking OpenAPI Callbacks
```

It's always "OpenAPI", and they will be in v4 and v5 too so we dont need to specify v3 for things. It'll just get wrong over time. :)

Reply:
That's right, fixed 👍